### ISSUE TITLE
Backface rendering parameters not working in interactive plots

### ISSUE THREAD

**Issue by user27182** [pyvista/pyvista#6079]
### Describe the bug, what's wrong, and what you expected.

See the docs here:
https://docs.pyvista.org/version/stable/examples/02-plot/backface_props.html

The backface params work for the static plots but not the interactive ones.

### Steps to reproduce the bug.

```python
Insert code here
```

### System Information

```shell
See docs.
```

**Comment by tkoyama010** [comment]
I guess it depends vtk.js is support it. @pyvista/kitware Any thought?

**Comment by jourdain** [comment]
I don't think that is supported within vtk.js but we've tested that one with WASM and it is translating fine.

**Comment by tkoyama010** [comment]
Excellent! Need to replace our document with a WASM version in the near future. It will solve this issue :)

**Comment by jourdain** [comment]
yes it will... But we are still working on it. Some work will be needed for the integration but it won't be too bad. And it will only be officially available for vtk>=9.4.

**Comment by KedoKudo** [comment]
just out of curiosity, is this issue still being actively worked on? Or maybe there is an alternative solution one can use while waiting for the long-term solution?
